templates/car_input.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <h1>Welcome to the Automotive Solution Tool!</h1>
+</head>
+<body>
+
+    <p>Please fill out the boxes below to search for your desired car:</p>
+
+    <form method="GET" action="get_car">
+        
+        <form>
```

Comment:
This form on line 15 will cause a blank form object to replace the get form we create above - line 11 is sufficient

Reply:
Line  is a <p> tag and I replaced it with a <h> tag. What exactly am I looking at? @Mr-3rd

Reply:
You must use the   `     to put code on the screen @pengxuemoua  for example            `<form method ="GET" action="get_car">`  Is the only tag you need. line 15 is duplicate, same with 23

Reply:
What I'm saying is:

`    <form method="GET" action ="get_user">
        <p>
            
            <label>GitHub Username:</label>
            <br>
            <input name='username'>
            <br>
            <button type="submit">Find Em!</button>

        </p>

    </form>`